jwql/website/apps/jwql/views.py

```diff
@@ -47,8 +48,7 @@
 from .data_containers import thumbnails
 from .data_containers import thumbnails_ajax
 from .forms import FileSearchForm
-from jwql.utils.utils import get_base_url, get_config, JWST_INSTRUMENTS, MONITORS, INSTRUMENTS_CAPITALIZED
```

Comment:
Looks like the line of `jwql.utils.utils` imports accidentally got deleted here.